src/bin/prepare.sources.sh

```
@@ -0,0 +1,4 @@
+#!/
+#!/usr/bin/bash
+RUST_BACKTRACE=full
```

Shellcheck (0.10.0)

[warning] RUST_BACKTRACE appears unused. Verify use (or export if used externally).

(SC2034)